policyengine_us/tests/policy/baseline/gov/states/mt/tax/income/subtractions/mt_old_age_subtraction.yaml

```diff
@@ -13,3 +13,27 @@
     state_code: MT
   output: 
     mt_old_age_subtraction: 5_500
+
+- name: MT joint filers both over 65 receive double old age subtraction
```

Comment:
can we add integration tests showing that the subtraction applies to the entire unit (head) when e.g. agi is attributed to head and deduction to spouse